scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
I agree with @cvrebert on this about that the smallest one ??? in his example should be `xs`. My suggestion for naming was based on that. I like and find the breakpoints from flexbox grid very user friendly. see https://github.com/kristoferjoseph/flexboxgrid/blob/master/src/css/flexboxgrid.css
notice `xs` has not explicit size definition also as @cvrebert suggests.